### PR TITLE
parameters for fjrod-granite fork added

### PR DESCRIPTION
**Pull request by ehsanhajian**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -164,6 +164,8 @@ ExecStart=/usr/local/bin/op-node \
   --metrics.enabled \
   --p2p.advertise.ip=<your-public-ip> \
   --metrics.port=7310 \
+  --override.fjord=1730106000 \
+  --override.granite=1730106000 \
```
Comment (ashutoshvarma) [inline README.md:168]: The timestamp is different from `sample.env`

File: minato/docker-compose.yml [hunk 2]
```diff
@@ -50,14 +50,16 @@ services:
       - OP_NODE_P2P_PEER_BANNING
       - METRICS_ENABLED
       - METRICS_PORT
+      - GETH_OVERRIDE_FJORD
+      - OP_NODE_OVERRIDE_GRANITE
```
Comment (ashutoshvarma) [inline minato/docker-compose.yml:54]: maybe name the envs in same convention
```
    - OP_NODE_OVERRIDE_FJORD
    - OP_NODE_OVERRIDE_GRANITE
```

File: minato/sample.env [hunk 3]
```diff
@@ -15,4 +15,6 @@ P2P_ADVERTISE_IP=<Node Public IP>
 OP_NODE_P2P_PEER_BANNING=false
 METRICS_ENABLED=true
 METRICS_PORT=7310
+GETH_OVERRIDE_FJORD=1729684800
```
Comment (PierreOssun) [inline minato/sample.env:18]: Even if it's a sample maybe better to add real value: `1730106000`

File: minato/docker-compose.yml (hunk 2 above)
Comment (ehsanhajian) [reply minato/docker-compose.yml:54]: I got this name from OP docs.

File: minato/sample.env (hunk 3 above)
Comment (ehsanhajian) [reply minato/sample.env]: Thanks. resolved
https://github.com/Soneium/soneium-node/pull/19/commits/081fd0c8c27f21e1972ed767c7dff79906c53915

File: README.md (hunk 1 above)
Comment (ehsanhajian) [reply README.md:168]: Thanks resolved.
https://github.com/Soneium/soneium-node/pull/19/commits/081fd0c8c27f21e1972ed767c7dff79906c53915

File: minato/docker-compose.yml (hunk 2 above)
Comment (ehsanhajian) [reply minato/docker-compose.yml]: https://github.com/Soneium/soneium-node/pull/19/commits/f2ab173801953d19d5740ce4ee22527f155ee6f6
please check

**Review by bLd75 — APPROVED** [review]
lgtm

**Pull request merged**